e2e-tests/cypress/e2e/journeys/insurance/account/sign-in/account-sign-in.spec.js

```diff
@@ -111,6 +111,10 @@ context('Insurance - Account - Sign in - I want to sign in into my UKEF digital
 
       field.input().should('exist');
     });
+
+    it('should render a revreal button that shows/reveals the password input', () => {
```

Comment:
'revreal'?

Reply:
Whoops! Good spot - i'll fix this immediately in another branch after merge.